lib/susedistribution.pm

```diff
@@ -272,14 +274,17 @@ sub init_consoles {
             });
     }
 
-    if (check_var('BACKEND', 's390x') || get_var('S390_ZKVM')) {
-        my $hostname = get_var('VIRSH_GUEST');
+    if (check_var('BACKEND', 'ipmi') || check_var('BACKEND', 's390x') || get_var('S390_ZKVM')) {
+        my $hostname;
+
+        $hostname = get_var('VIRSH_GUEST') if get_var('S390_ZKVM');
+        $hostname = get_required_var('SUT_IP') if check_var('BACKEND', 'ipmi');
```

Comment:
Just a thought - should we unify on using SUT_IP consistantly instead of 3 variables for the same job (SUT_IP, VIRSH_GUEST, and S390_HOST)?